src/framework/mlt_property.h

```diff
@@ -30,7 +30,7 @@
 #include <sys/param.h>
 #endif
 
-#if defined(__GLIBC__) && !defined(__APPLE__)
+#if defined(__GLIBC__) && !defined(__APPLE__) || HAVE_LOCALE_H
```

Comment:
This should have parenthesis to clarify the order of logical operations. && is higher precedence than ||:
`#if (defined(__GLIBC__) && !defined(__APPLE__)) || HAVE_LOCALE_H`